helm-chart/renku/templates/secrets-storage/secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "renku.fullname" . }}-secrets-storage
```

Comment:
What is the difference between this secret and the one generated by the init script? Why dont we generate all of them with the init script?

Reply:
this one is for symmetric encryption at rest in the DB. but yes, it should also be in platform init, it's a leftover from the first iteration of the service.